estail.go

```diff
@@ -18,23 +18,43 @@ func fatalf(msg string, args ...interface{}) {
 	os.Exit(2)
 }
 
+// StringArray implements flag.Value interface
+type StringArray []string
+
+func (a *StringArray) Set(s string) error {
+	msgs := strings.Split(s, ",")
+	*a = append(*a, msgs...)
+	return nil
+}
+
+func (a *StringArray) String() string {
+	return strings.Join(*a, ",")
+}
+
 func main() {
 	host := "localhost:9200"
 	indexPrefix := "logstash-"
-	msgField := "@message"
+	msgFields := StringArray{}
 	timeField := "@timestamp"
 	exclude := ""
 	size := 1000
+	poll := 1
 
 	flag.StringVar(&host, "host", host, "host and port of elasticsearch")
 	flag.StringVar(&indexPrefix, "prefix", indexPrefix, "prefix of log indexes")
-	flag.StringVar(&msgField, "message", msgField, "message field to display")
+	flag.Var(&msgFields, "message", "message fields to display")
 	flag.StringVar(&timeField, "timestamp", timeField, "timestap field to sort by")
 	flag.StringVar(&exclude, "exclude", exclude, "comma separated list of field:value pairs to exclude")
 	flag.IntVar(&size, "size", size, "number of docs to return per polling interval")
+	flag.IntVar(&poll, "poll", poll, "time in seconds to poll for new data from ES")
```

Comment:
A StringVar that gets turned into a Duration with `time.ParseDuration` might be nice, but this is fine.